diff --git a/build/styles.js b/build/styles.js
index fc05d14a36074305d962fe5debf1f7bd78929b26..44460dc46fbcfe151969348dc0e6ca8a1dace435 100644
--- a/build/styles.js
+++ b/build/styles.js
@@ -1,10 +1,16 @@
 import Yoga from 'yoga-layout';
-const positionEdges = [
-    ['top', Yoga.EDGE_TOP],
-    ['right', Yoga.EDGE_RIGHT],
-    ['bottom', Yoga.EDGE_BOTTOM],
-    ['left', Yoga.EDGE_LEFT],
-];
+let _positionEdges;
+function positionEdges() {
+    if (!_positionEdges) {
+        _positionEdges = [
+            ['top', Yoga.EDGE_TOP],
+            ['right', Yoga.EDGE_RIGHT],
+            ['bottom', Yoga.EDGE_BOTTOM],
+            ['left', Yoga.EDGE_LEFT],
+        ];
+    }
+    return _positionEdges;
+}
 const applyPositionStyles = (node, style) => {
     if ('position' in style) {
         let positionType = Yoga.POSITION_TYPE_RELATIVE;
@@ -16,7 +22,7 @@ const applyPositionStyles = (node, style) => {
         }
         node.setPositionType(positionType);
     }
-    for (const [property, edge] of positionEdges) {
+    for (const [property, edge] of positionEdges()) {
         if (!(property in style)) {
             continue;
         }
